plugins/modules/dcnm_links.py

```diff
@@ -3076,7 +3076,8 @@ def dcnm_links_compare_intra_fabric_link_params(self, wlink, hlink):
             )
         ):
             if (
-                str(wlink["nvPairs"]["ENABLE_MACSEC"]).lower()
+                hlink["nvPairs"].get("ENABLE_MACSEC") and
```

Comment:
I understand updating to use a safe `get` but why the additional `and` check here?